.github/workflows/reusable-build.yml

```diff
@@ -191,13 +191,13 @@ jobs:
         tar -xf ..\..\x64-${{ matrix.configurations }}-cilium-xdp.zip
 
     - name: Zip Build Output
-      if: steps.skip_check.outputs.should_skip != 'true'
+      if: (success() || failure()) && (steps.skip_check.outputs.should_skip != 'true')
```

Comment:
this won't zip the build output if the job is canceled, is that what we want?

Reply:
We want to get the zip if the build fails. 
Does 'cancel' mean the build has failed? (or some time limit if it exceeds?)

Reply:
cancel means either it failed due to time limit exceeded, or manual cancel.

Reply:
Thanks. In that case, it will be nice to have 'cancel' due to the time limit being exceeded (if there is a time limit set)
Can you suggest what flag/call/syntax to add in addition?

Reply:
`always()` instead of `success() || failure()` should cover the cancellation scenario also.

Reply:
Thanks. Changed to always.